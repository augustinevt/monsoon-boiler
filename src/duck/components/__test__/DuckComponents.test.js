import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount } from 'enzyme';

import DuckComponent from '../DuckComponent';

test('DuckComponent Snapshot', () => {
  const snapshot = renderer.create(
    <DuckComponent type="stream" foo={'Skrit Skrit'} />
  ).toJSON();

  expect(snapshot).toMatchSnapshot();
});

xtest('Duck Enzyme', () => {
  const wrapper = mount(<DuckComponent foo={'Skrit Skrit'}/>);

  const thing = wrapper.find('div');
  // expect(wrapper.find('div')).toHaveLength(1);
});
