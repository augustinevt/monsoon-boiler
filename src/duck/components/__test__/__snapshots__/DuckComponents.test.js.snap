// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DuckComponent Snapshot 1`] = `
<div
  className="duck-component"
>
  <div
    className="button default-button"
    onClick={[Function]}
  >
    Open Form
  </div>
</div>
`;
